packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -192,6 +193,7 @@ const VisitHeader: React.FC = () => {
             <PatientInfo patient={patient} />
           </div>
           <HeaderGlobalBar>
+            <ExtensionSlot extensionSlotName="visit-header-right-slot" />
```

Comment:
```suggestion
            <ExtensionSlot name="visit-header-right-slot" />
```

Reply:
[`extensionSlotName`](https://github.com/openmrs/openmrs-esm-core/blob/43a4cbe41f05df9e828d4a30c9d94b88f46a18ff/packages/framework/esm-react-utils/src/ExtensionSlot.tsx#L9) is deprecated. Use `name` instead.